package.json

```diff
@@ -6,12 +6,18 @@
   "scripts": {
     "start": "node run-server.js",
     "test": "./node_modules/.bin/babel-node node_modules/mocha/bin/_mocha --recursive -R spec",
-    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec"
+    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec",
```

Comment:
You don't need to fully qualify the path of a local module. npm will automatically create a global symbol local bin files while running scripts